File: Android/tuilivekit/src/main/java/com/trtc/uikit/livekit/state/operation/OperationState.java
package com.trtc.uikit.livekit.state.operation;

public class OperationState {
    public RoomState       roomState       = new RoomState();
    public SeatState       seatState       = new SeatState();
    public UserState       userState       = new UserState();
    public MediaState      mediaState      = new MediaState();
    public BeautyState     beautyState     = new BeautyState();
    public ConnectionState connectionState = new ConnectionState();

    public void reset() {
        roomState.reset();
        seatState.reset();
        userState.reset();
        mediaState.reset();
        beautyState.reset();
        connectionState.reset();
    }
}

File: Flutter/livekit/lib/live_stream/state/operation/operation_state.dart
import 'index.dart';

class OperationState {
  final RoomState roomState = RoomState();
  final SeatState seatState = SeatState();
  final UserState userState = UserState();
  final MediaState mediaState = MediaState();
  final BeautyState beautyState = BeautyState();

  void reset() {
    roomState.reset();
    seatState.reset();
    userState.reset();
    mediaState.reset();
    beautyState.reset();
  }
}
